bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,154 @@
+
+        <a href="https://blog.mozilla.org/en/firefox/" class="is-default mzp-c-button mzp-t-lg" data-cta-type="button" data-cta-text="Learn more" data-cta-position="primary">Learn more</a>
+
+        {{ download_firefox_thanks(alt_copy='Download Firefox', dom_id='protocol-nav-download-firefox', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='primary') }}
+      </div>
+    </div>
+  </header>
+  <main class="c-page-main-content">
+    <section>
+    <div class="c-browser-windows">
+      <div class="c-layered-browser">
+        {% call browser_border(class='c-browser-window-top-wrapper mzp-l-content mzp-t-content-md mzp-t-content-nospace') %}
+        <div class="c-browser-window-top">
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+
+        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser yellow mzp-l-content mzp-t-content-md mzp-t-content-nospace') %}
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+
+        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser dark-yellow mzp-l-content mzp-t-content-md mzp-t-content-nospace') %}
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+
+        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser darker-yellow mzp-l-content mzp-t-content-md mzp-t-content-nospace') %}
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+      </div>
+
+        {% call browser_border(class='mzp-l-content mzp-t-content-md mzp-t-content-nospace', heading='Nothing Personal') %}
+          <div class="c-browser-window-top">
+            <h4>You’re great — probably.</h4>
+            <p>We just don’t need a lot of your data to be a <strong>fast, reliable</strong> browser that keeps up with the big guys.</p>
+          </div>
+        {% endcall %}
+
+        <div class="c-thug-life-gif-wrapper mzp-l-content mzp-t-content-md mzp-t-content-nospace">
+          <div class="c-thug-life-gif c-detached-gif">
+            <img loading="lazy" src="{{ static('img/firefox/nothing-personal/thug-life.gif') }}" alt="" class="c-gif">
+            <img loading="lazy" src="{{ static('img/firefox/nothing-personal/thug-life-img.jpg') }}" alt="" class="c-image">
+          </div>
```

Comment:
**issue:** this should be a `<picture>` element otherwise the browser will download both images when only one is needed